framework/include/meshgenerators/CombinerGenerator.h

```diff
@@ -40,9 +43,15 @@ class CombinerGenerator : public MeshGenerator
   /// The mesh generators to use
   const std::vector<MeshGeneratorName> & _input_names;
 
-  /// The (offsets) positions for each mesh
+  /// The (offsets) positions for each mesh if specified as a vector of points
   const std::vector<Point> & _positions;
 
+  /// The (offsets) positions for each mesh if specified as vector of points in file
+  const std::vector<FileName> & _positions_file;
+
+  /// The (offset) positions for each mesh
+  std::vector<Point> _generator_positions;
```

Comment:
For simplicity, I say just keep the single `_positions` variable. If you want to keep it const, you could make `fillPositions()` just return a vector depending on the parameter combo.

Reply:
hmm, but if `getParam` (if user does `positions = '1 1 1'`) requires `const`, and `push_back` (if user does `positions_file = 'positions.txt'`) won't allow `const`, is that possible?

Reply:
Nevermind, ignore my brain lapse here...